src/scss/theme.scss

```diff
@@ -279,7 +430,7 @@ body {
 
 /* Nav: hide second level (shown on .active) */
 
-.bd-toc .nav {
+.uu-toc .nav {
```

Comment:
I think it is this rename that makes that the javascript is not working anymore. In the scrollspy function, we query selectors on "#bd-toc-nav".

Reply:
Hi Joris, good remark on this typo. No clear idea how this ended up  in the PR. The html selector should not have changed, only styling changed I think. Fixed now.